src/github.ts

```diff
@@ -5,6 +5,46 @@ import {Octokit} from '@octokit/rest'
 const token: string = core.getInput('token', {required: true})
 const octokit = new Octokit({auth: token})
 
+export async function changeUserPermissions(
+  org: string,
+  repo: string,
+  user: string,
+  permission: string
+): Promise<void> {
+  try {
+    const res = await octokit.request(
+      'PUT /repos/{owner}/{repo}/collaborators/{username}',
+      {
+        owner: org,
+        repo,
+        username: user,
+        permission
+      }
+    )
+    if (res.status === HTTP.CREATED) {
+      core.debug(`New collaborator invitation created for user ${user}`)
```

Comment:
The README update mentions the functionality to send invite to user was removed. 
The `changeUserPermissions` function mentions a scenario where it could send an invite.

Reading through the code it seems that if `checkUser` is `false` and `promoteUser` is `true` then a user requesting a fork that isn't a member of the org would be invited as a collaborator with admin permissions on the fork. This may or may not be the desired outcome. If it is the desired outcome we should document that scenario.
If it's not the desired outcome we'll want to prevent the invitation scenario in `changeUserPermissions`

Reply:
Good catch -- the "collaborator invitation" line was taken straight from the GitHub API's docs for the `/collaborators` endpoint. It is misleading, because the invite in this case would be repository-specific, and there's much lower stakes if the user never accepts the invite, since they'll still have access to the fork by virtue of their `/wayfair-contribs` org membership. I can update the debug line to be more specific about which type of invite is being sent here.

For your second comment -- you are correct that with the current behavior, that scenario could allow for a user to receive elevated permissions without an org check. Given that organizations are likely to be the only ones using the permissions feature (since if you're making forks on your own account, you can give yourself whatever perms you want), it's arguable that we should only allow `promoteUser` in combination with `checkUser` passing successfully. Agree that the condition should be made explicit in the `README` regardless of which option we go with!